Traceback (most recent call last):
  File "c:\Users\S-FAMILY\Desktop\run\SAZOM.py", line 6, in <module>
    from rembg import remove
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\rembg\__init__.py", line 5, in <module>
    from .bg import remove
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\rembg\bg.py", line 7, in <module>  
    from cv2 import (
ule>
    bootstrap()
strap
    native_module = importlib.import_module("cv2")
PS C:\Users\S-FAMILY\Desktop\run> & C:/Users/S-FAMILY/AppData/Local/Programs/Python/Python39-32/python.exe c:/Users/S-FAMILY/Desktop/run/SAZOM.py
Traceback (most recent call last):
  File "c:\Users\S-FAMILY\Desktop\run\SAZOM.py", line 6, in <module>
    from rembg import remove
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\rembg\__init__.py", line 5, in <module>
    from .bg import remove
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\rembg\bg.py", line 7, in <module>  
    from cv2 import (
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\cv2\__init__.py", line 181, in <module>
    bootstrap()
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\site-packages\cv2\__init__.py", line 153, in bootstrap
    native_module = importlib.import_module("cv2")
  File "C:\Users\S-FAMILY\AppData\Local\Programs\Python\Python39-32\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: DLL load failed while importing cv2: %1 is not a valid Win32 application.
